addons/arsenal/functions/fnc_verifyLoadout.sqf

```diff
@@ -43,8 +43,14 @@ private _fnc_filterLoadout = {
                 if !(_name in GVAR(virtualItemsFlat)) then {
                     _name = _name call FUNC(baseWeapon);
                     if !(_name in GVAR(virtualItemsFlat)) then {
-                        _unavailableItemsList pushBack _name;
-                        _name = "";
+                        // This could be a backpack
+                        private _temp = [_name, "CfgVehicles"] call CBA_fnc_getNonPresetClass;
+                        if (_temp == "") then { // It's not
+                            _unavailableItemsList pushBack _name;
+                            _name = "";
+                        } else {
+                            _name = _temp;
```

Comment:
Don't we have to re-verify if the non preset classed backpack is part of the arsenal?